api/tests/unit/application/target-profiles/index_test.js

```diff
@@ -250,6 +295,10 @@ describe('Integration | Application | Target Profiles | Routes', () => {
 
     it('should exist', async () => {
       // given
+      sinon.stub(securityPreHandlers, 'checkUserHasRolePixMaster').callsFake((request, h) => h.response(true));
+      sinon.stub(targetProfileController, 'updateTargetProfileName').callsFake((request, h) => h.response('ok').code(204)); const httpTestServer = new HttpTestServer();
```

Comment:
```suggestion
      sinon.stub(targetProfileController, 'updateTargetProfileName').callsFake((request, h) => h.response('ok').code(204));
      const httpTestServer = new HttpTestServer();
```

**Nitpick : ** Ajout d'un saut de ligne avant la déclaration du serveur.